frontend/src/metabase/query_builder/components/chart-type-selector/ChartTypeList/ChartTypeList.tsx

```diff
@@ -0,0 +1,36 @@
+import { Grid } from "metabase/ui";
+
+import { ChartTypeOption, type ChartTypeOptionProps } from "../ChartTypeOption";
+
+export type ChartTypeListProps = {
+  visualizationList: ChartTypeOptionProps["visualizationType"][];
```

Comment:
nit: using just underlying types  like `CardDisplayType` here would be a bit more readable